assets/javascripts/discourse/services/chat.js

```diff
@@ -60,6 +60,22 @@ export default class Chat extends Service {
 
   @and("currentUser.has_chat_enabled", "siteSettings.chat_enabled") userCanChat;
 
+  @computed("currentUser.staff", "currentUser.groups.[]")
+  get userCanDirectMessage() {
+    if (!this.currentUser) {
+      return false;
+    }
+
+    return (
+      this.currentUser.staff ||
+      this.currentUser.isInAnyGroups(
+        (this.siteSettings.direct_message_enabled_groups || "11") // trust level 1 auto group
```

Comment:
This is mainly a workaround for the limitations of JS tests loading settings, this setting should never be null IRL. This PR in core is working on the solution but I don't want to hold up this chat feature for it https://github.com/discourse/discourse/pull/18413